Source/Config.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SimpleRefuel {
+	public class Config {
+		private const string _configFilePath = "GameData/SimpleRefuel/SimpleRefuel.cfg";
```

Comment:
Underscores should be used for temporary variables only. It is hardcoded anyway, so placing the literal string into the getter of ConfigFilePath makes more sense.